### ISSUE TITLE
Issue with "<iframe>": Exemples redirect to dead page

### ISSUE THREAD

**Issue by Onion2222** [mdn/content#5541]
MDN URL: https://developer.mozilla.org/fr/docs/Web/HTML/Element/iframe

#### What information was incorrect, unhelpful, or incomplete?
Exemple dosen't work, they redirect to dead page.

#### Specific section or headline?
Exemples.

#### What did you expect to see?
A real example.

#### Did you test this? If so, how?
Yes, I tried to use the exemple.

<details>
<summary>MDN Content page report details</summary>

* Folder: `fr/web/html/element/iframe`
* MDN URL: https://developer.mozilla.org/fr/docs/Web/HTML/Element/iframe
* GitHub URL: https://github.com/mdn/translated-content/blob/main/files/fr/web/html/element/iframe/index.html
* Last commit: https://github.com/mdn/translated-content/commit/944d0fe9773731bb4952fd986456a5140e490be3
* Document last modified: 2021-05-31T02:16:49.000Z

</details>

**Comment by dorward** [comment]
My first reading of this was that it was wrong as the example at the top of the page was working. The one in the Examples selection does error though. The English version of this document uses example.org instead.

**Comment by dorward** [comment]
Looking more closely there appear to be a couple of different problems. 

* The examples use https://mdn-samples.mozilla.org which doesn't appear to be live at present
* The examples are hosted on https://yari-demos.prod.mdn.mozit.cloud which is outputting `x-frame-options
	DENY` headers

**Comment by sideshowbarker** [comment]
> Looking more closely there appear to be a couple of different problems.
> 
> * The examples use [mdn-samples.mozilla.org](https://mdn-samples.mozilla.org) which doesn't appear to be live at present

Yes — mdn-samples.mozilla.org has been decommissioned. The broken links to it are a known issue. But I don’t have the issue number at hand, and not sure anybody’s actively working on a fix.

> * The examples are hosted on [yari-demos.prod.mdn.mozit.cloud](https://yari-demos.prod.mdn.mozit.cloud) which is outputting `x-frame-options DENY` headers

I think that is also a known issue — but also don’t have the issue number at hand, nor know if somebody is working on fixing it.

**Comment by teoli2003** [comment]
cc/ @SphinxKnight that is one of the leads fo the French localization and will know for sure.

**Comment by SphinxKnight** [comment]
Thanks @teoli2003 for the ping and @dorward for the report. I'm moving this one over to mdn/translated-content and will provide an update soon-ish to fix this.
